llvm/test/SYCLLowerIR/esimd_global_crash.ll

```diff
@@ -0,0 +1,23 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -ESIMDLowerVecArg -S | FileCheck %s
+
```

Comment:
please add comment what this test checks

Reply:
fixed.